apps/service_providers/migrations/0019_auto_20241029_0901.py

```diff
@@ -0,0 +1,222 @@
+# Generated by Django 5.1.2 on 2024-10-28 17:35
+import logging
+
+from collections import namedtuple
+from django.db import migrations
+from django.db.models import Q
+
+logger = logging.getLogger(__name__)
+
+
+def _create_llm_provider_models(apps, schema_editor):
+    _create_default_llm_provider_models(apps, schema_editor)
+    _create_custom_llm_provider_models(apps, schema_editor)
+
+DefaultLlmProviderModel = namedtuple('DefaultLlmProviderModel', ["type", "name", "max_token_limit"])
+
+DEFAULT_LLM_PROVIDER_MODELS = {
+    DefaultLlmProviderModel("openai", "gpt-4o-mini", 8192),
+    DefaultLlmProviderModel("anthropic", "claude", 8192),
+}
+
+def _create_default_llm_provider_models(apps, schema_editor):
+    LlmProviderModel = apps.get_model("service_providers", "LlmProviderModel")
+    for default_provider_model in DEFAULT_LLM_PROVIDER_MODELS:
+        LlmProviderModel.objects.create(
+            team=None,
+            type=default_provider_model.type,
+            name=default_provider_model.name,
+            max_token_limit=default_provider_model.max_token_limit,
+        )
+    # TODO: Get list of current global models
```

Comment:
Maybe from here: https://raw.githubusercontent.com/FlowiseAI/Flowise/main/packages/components/models.json

Reply:
https://github.com/dimagi/open-chat-studio/pull/810/commits/ed258a38cf4ca9aa6ffe7313bb3d6ac27b5d6d40

I removed the "vision" ones, but kept all the others.